test/Microsoft.AspNet.Mvc.Xml.Test/XmlAssert.cs

```diff
@@ -20,30 +22,62 @@ public static class XmlAssert
         /// <param name="actualXml">Actual xml string.</param>
         public static void Equal(string expectedXml, string actualXml)
         {
-            var sortedExpectedXDoc = SortAttributes(XDocument.Parse(expectedXml));
```

Comment:
nit: add an early exit that returns `true` when `string.Equals(expectedXml, actualXml, StringComparison.Ordinal)`